operator/internal/handlers/lokistack_create_or_update_test.go

```diff
@@ -221,6 +228,10 @@ func TestCreateOrUpdateLokiStack_SetsNamespaceOnAllObjects(t *testing.T) {
 			k.SetClientObject(out, &defaultGatewaySecret)
 			return nil
 		}
+		if apiServer.Name == name.Name {
+			k.SetClientObject(out, &apiServer)
+			return nil
+		}
```

Comment:
Why do we need for the general usecase the `openshiftv1.APIServer` CR here? This should work without it.